crates/turbopack-ecmascript/src/transform/mod.rs

```diff
@@ -134,6 +139,18 @@ impl EcmascriptInputTransform {
             ..
         } = ctx;
         match self {
+            EcmascriptInputTransform::GlobalTypeofs { window_value } => {
+                let mut typeofs: AHashMap<JsWord, JsWord> = Default::default();
+                let val = window_value.to_owned();
+                typeofs.insert("window".into(), JsWord::from(val));
```

Comment:
```suggestion
                typeofs.insert("window".into(), JsWord::from(&**window_value));
```

You don't need to create a `String` to create `JsWord`